plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/storage/KVMStorageProcessor.java

```diff
@@ -1021,7 +1032,9 @@ public Answer backupSnapshot(final CopyCommand cmd) {
                 command.add(NAME_OPTION, snapshotName);
                 command.add("-p", snapshotDestPath);
 
-                descName = UUID.randomUUID().toString();
+                if (isCreatedFromVmSnapshot) {
```

Comment:
can you please explain why we had to keep this specific check for vmsnapshot case ?

Reply:
Unfortunately, I did not note this in the commit for this and don't recollect exactly what it was addressing.  I do not see reason this is needed for the updates for this PR and will update/remove this change to avoid any regression.